R-Project/R/fpRerF.R

```diff
@@ -78,6 +85,24 @@ fpRerF <-
 		forest_module$setParameterDouble("mtryMult",mtryMult)
 		}
 
+		if(forestType == "binnedBaseTern"){
```

Comment:
it isn't obvious to me why we require binnedBaseTern in order to use s-rerf...

Reply:
... if binnedBaseRerF is specified, we set `methodToUse=1`.  if s-rerf -- `methodToUse=2`?

Reply:
Good idea.